### PR TITLE
[IMPROVE] Remove 'Livechat_open_inquiery_show_connecting' setting

### PR DESCRIPTION
**Pull request by renatobecker**
Related to -> https://github.com/RocketChat/Rocket.Chat.Livechat/pull/210.

We decided to remove the "Livechat_open_inquiery_show_connecting" setting that was used to prevent livechat users from sending messages while they were waiting in the queue.
Now, if the routing method = 'Guest Pool', the livechat users will be able to send messages even when they are waiting for an available agent.

On the new client, an alert will be displayed, notifying the users that they are waiting in the queue.

### CODE REVIEW

**Comment by sampaiodiego** [comment]
the default value is already `false`, I don't get why remove it..

just remembering this is added by @AlecTroemel in https://github.com/RocketChat/Rocket.Chat/pull/4572 I wonder what was his use case for it.

**Comment by AlecTroemel** [comment]
I believe my intent was to inform the client they were "in the pool", ie waiting to connect to an agent. Otherwise, they could assume they were connected to somebody, start sending messages and get frustrated with no response. I think the clients status in the pool should be communicated to them is some way... though I don't use the live chat feature of rocket chat anymore so it's just my 2 cents. Hope that helps!

**Comment by renatobecker** [comment]
> the default value is already `false`, I don't get why remove it..
> 
@engelgabriel required to remove the setting since it will not be used anymore..

**Comment by renatobecker** [comment]
> the default value is already `false`, I don't get why remove it..
> 
> just remembering this is added by @AlecTroemel in #4572 I wonder what was his use case for it.

**Comment by renatobecker** [comment]
> the default value is already `false`, I don't get why remove it..
> 
@engelgabriel required to remove the setting since it will not be used anymore..

**Comment by renatobecker** [comment]
Here's how the widget will look when waiting for an available agent:

55252716-48848e80-5232-11e9-8051-16e9682cb686

**Comment by renatobecker** [comment]
> can you please write a migration to remove the setting `Livechat_open_inquiery_show_connecting` as well?
> 
> please remove the `i18n` key too.
> 
> @renatobecker how will it look on the old client? since it is still the default, I think it should have some status as you showed for the new client.

Okay, I'll create a migration to remove the deprecated setting, as well as removing the `i18n` keys..
Regarding the old client, I'll add some alert as we do on the new client.

**Comment by renatobecker** [comment]
@sampaiodiego,

I Just pushed the code adding an alert when the guest is waiting for an available agent, as displayed below(Old livechat version):

Screen Shot 2019-04-17 at 15 45 46

File: packages/rocketchat-livechat/.app/i18n/en.i18n.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
   "Please_answer_survey": "Please take a moment to answer a quick survey about this chat",
   "Please_choose_a_department": "Please choose a department",
   "Please_fill_name_and_email": "Please fill name and email",
+  "Please_wait_for_the_next_available_agent": "Please, wait for the next available agent..",
```
Comment (sampaiodiego) [inline packages/rocketchat-livechat/.app/i18n/en.i18n.json]:
```suggestion
  "Please_wait_for_the_next_available_agent": "Please, wait for the next available agent.",
```

File: packages/rocketchat-livechat/.app/i18n/pt-BR.i18n.json [hunk 2]
```diff
@@ -24,6 +24,7 @@
   "Please_answer_survey": "Por favor, tire um momento para responder uma rápida pesquisa sobre este chat",
   "Please_choose_a_department": "Por favor escolha um departamento",
   "Please_fill_name_and_email": "Por favor, preencha nome e email",
+  "Please_wait_for_the_next_available_agent": "Por favor, aguarde o próximo agente disponível..",
```
Comment (sampaiodiego) [inline packages/rocketchat-livechat/.app/i18n/pt-BR.i18n.json]:
```suggestion
  "Please_wait_for_the_next_available_agent": "Por favor, aguarde o próximo agente disponível.",
```

**Comment by rasos** [comment]
There is a typo in the german translation, I corrected it in Lingohub but can't find that term in `de.i18n.json`

> "Please_wait_for_the_next_available_agent": "Bitte auf die nächste freie Beratung warten", 

I also tried to apply a manual translation but it seems the livechat gets his translations from somewhere else?

**Comment by sampaiodiego** [comment]
thanks @rasos .. the new livechat widget has its own translation string at https://github.com/RocketChat/Rocket.Chat.Livechat 

we should be creating a lingohub project for it soon.

**Comment by rasos** [comment]
Ah I see, the translations in the Livechat 2.0 project seem okay. 

However we may be keeping the 1.0 widget for a while, we prefer the bar and not the icon. Any hints where to find that? github does not know e.g. `Please_wait_for_the_next_available_agent` in any RocketChat project and it seems not to be taken from Lingohub.

**Pull request merged**